Content.Tests/DMProject/Tests/Math/minmax.dm

```diff
@@ -0,0 +1,16 @@
+/proc/RunTest()
+	ASSERT(min(1,2,3) == 1)
+	ASSERT(min(null, 1) == null)
+	ASSERT(min(null, -1) == -1)
+	ASSERT(min(1, null) == null)
+	ASSERT(min(0,null)==null)
+	ASSERT(min("a","b","c")=="a")
+	ASSERT(min("b","a","c")=="a")
+
+	ASSERT(max(1,2,3) == 3)
+	ASSERT(max(null, 1) == 1)
+	ASSERT(max(null, -1) == null)
+	ASSERT(max(1, null) == 1)
+	ASSERT(max(0,null)==null)
```

Comment:
This does not test `max(null, 0)`. This PR gives `0` which is the same as BYOND.